cumulusci/tasks/bulkdata/base_generate_data_task.py

```diff
@@ -82,8 +82,9 @@ def init_db(db_url, mappings):
         metadata = MetaData()
         metadata.bind = engine
         if mappings:
-            for mapping in mappings.values():
-                create_table(mapping, metadata)
+            for name, mapping in mappings.items():
+                if "table" in mapping and mapping["table"] not in metadata.tables:
+                    create_table(mapping, metadata)
```

Comment:
Yes. It would be nicer to check but we don't anywhere else in our mapping file infrastructure. (e.g. Load.py doesn't check that the mapping file and the database correspond) Baby steps...